<launch>
    <node pkg="rknn_pkg" type="unet_ros_node.py" name="unet_ros_node" output="screen"/>
</launch>